### PR TITLE
mention the --build-filters flag in the command line options section

### PR DESCRIPTION
**Pull request by jakemac53**
Replacement PR for https://github.com/dart-lang/build/pull/3367. We now just link the partial builds doc which explains the feature more thoroughly.

### CODE REVIEW

File: build_runner/README.md [hunk 1]
```diff
@@ -85,6 +85,9 @@ All the above commands support the following arguments:
   be provided.
 - `--[no-]fail-on-severe`: Whether to consider the build a failure on an error
   logged. By default this is false.
+- `--build-filter`: Build filters allow you to choose explicitly which files to
+  build instead of building entire directories. See further documentation on
+  this feature [here](../docs/partial_builds.md).
```
Comment (natebosch) [inline build_runner/README.md:90]: Will this link work when the readme is on pub?

**Review by natebosch — APPROVED** [review]

File: build_runner/README.md (hunk 1 above)
Comment (jakemac53) [reply build_runner/README.md:90]: Huh, good question. cc @jonasfj

Comment (natebosch) [reply build_runner/README.md:90]: We might want to use an absolute URL here to be safe.

Comment (jakemac53) [reply build_runner/README.md:90]: Yeah ok I will just make it absolute, the relative url works better with IDEs (navigates to the actual file in your IDE), but the absolute uri still takes you to a good place.

**Pull request merged**